demos/TransferLearning/README.md

```diff
@@ -25,9 +31,9 @@ jupyter notebook TransferLearningDemo.ipynb
 
 ## Running the demo through the command line
 ```
-usage: TransferLearningDemo [-h] [--gui] [--training]
+usage: TransferLearningDemo [-h] [--gui] [--training] 
```

Comment:
Not sure I understand the new architecture, but wasn't the `--gui` argument dropped?